### DISCUSSION TITLE
Missing images

### DISCUSSION THREAD

**Discussion by tumblwd** [GrandyB/aoe2buildorders#2] [General]
It looks like images for Castle and Bow Saw are missing from the buildings and technology folders.  Build orders are using the alt text for now.

**Comment by tumblwd** [comment]
I'm also not sure what to do with generic unique unit image.  I have called it 'UU' for now.

**Comment by GrandyB** [comment]
Have added all the above.

uniqueunit
uniquetech_castle
uniquetech_imperial
bowsaw

For the 'many' part, you can use emojipedia.com and grab the infinity sign: ♾️
Am editing your submissions post-merge to use this!

**Reply by tumblwd** [reply]
Thanks.  Did not know about the infinity so that will be really useful.
